GCTC-NTGC/gc-digital-talent
# Feature - Add support API endpoint

Resolves #4187.

h/t @esizer for co-piloting.

### Steps to test

1. Create a test [Freshdesk account](https://freshdesk.com/signup-free)
2. Add environment variables to api/.env based on api/.env.example from test Freshdesk account
3. Clear config cache `php artisan config:cache` (inside shell)
4. Clear route cache `php artisan route:cache` (inside shell)
5. Build talent search `npm run production --workspace=talentsearch`
6. Run talent search storybook `npm run storybook --workspace=talentsearch`
7. Verify successful ticket creation
8. Verify unsuccessful ticket creation (by not sending in one of the [required parameters](https://developers.freshdesk.com/api/#create_ticket) to Freshdesk)
